timemachine/fe/reweighting.py

```diff
@@ -0,0 +1,286 @@
+__all__ = [
+    "one_sided_exp",
+    "construct_endpoint_reweighting_estimator",
+    "construct_mixture_reweighting_estimator",
+    "interpret_as_mixture_potential",
+]
+
+from typing import Callable, Collection
+
+from jax import numpy as np
+from jax.scipy.special import logsumexp
+
+Samples = Params = Collection
+Energies = Array = np.array
```

Comment:
```suggestion
Energies = Array = np.ndarray
```
(without this I get `error: Function "jax._src.numpy.lax_numpy.array" is not valid as a type`)